src/icons/words.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3 18C2.44772 18 2 17.5523 2 17C2 16.4477 2.44772 16 3 16H14C14.5523 16 15 16.4477 15 17C15 17.5523 14.5523 18 14 18H3ZM3 13C2.44772 13 2 12.5523 2 12C2 11.4477 2.44772 11 3 11H21C21.5523 11 22 11.4477 22 12C22 12.5523 21.5523 13 21 13H3ZM3 8C2.44772 8 2 7.55228 2 7C2 6.44772 2.44772 6 3 6H21C21.5523 6 22 6.44772 22 7C22 7.55228 21.5523 8 21 8H3Z" fill="#00344D"/>
```

Comment:
```suggestion
<path d="M3 18C2.44772 18 2 17.5523 2 17C2 16.4477 2.44772 16 3 16H14C14.5523 16 15 16.4477 15 17C15 17.5523 14.5523 18 14 18H3ZM3 13C2.44772 13 2 12.5523 2 12C2 11.4477 2.44772 11 3 11H21C21.5523 11 22 11.4477 22 12C22 12.5523 21.5523 13 21 13H3ZM3 8C2.44772 8 2 7.55228 2 7C2 6.44772 2.44772 6 3 6H21C21.5523 6 22 6.44772 22 7C22 7.55228 21.5523 8 21 8H3Z" fill="currentColor"/>
```

This isn't *technically* necessary when using it through `astro-icon`, but would be nice in case this icon ends up elsewhere :)